nl/guides/for-students/exercises/README.md

```diff
@@ -0,0 +1,199 @@
+:::
+
+Na het [indienen](#indienen-van-een-oplossing) van een [oplossing](#navigeren-naar-een-oplossing) wordt automatisch de tab `Oplossingen` geselecteerd. Deze tab bevat een overzicht van alle oplossingen die je in de [cursus]() hebt ingediend voor de [oefening](#navigeren-naar-een-oefening). Deze oplossingen worden in het overzicht opgelijst in omgekeerde chronologische volgorde (meest recente bovenaan), waardoor de oplossing die je net hebt ingediend helemaal bovenaan staat. Het overzicht bevat voor elke oplossing het [tijdstip](#tijdstip) van indienen, de [status](#status) en een korte [samenvatting](#samenvatting) van de [feedback](#interpreteren-van-feedback). In het overzicht zie je vóór elke oplossing ook een [icoontje](../courses/#indienstatus) dat correspondeert met de status van de oplossing.
+
+<h1 id="wachtrij"/>
+
+Om overbelasting van het platform tegen te gaan, worden [oplossingen](#navigeren-naar-een-oplossing) niet onmiddellijk beoordeeld na het [indienen](#indienen-van-een-oplossing) maar worden ze eerst in een **wachtrij** geplaatst. Zolang een oplossing in de wachtrij staat heeft ze de [status](#status) `In de wachtrij...`. Van zodra het platform klaar is om een oplossing te beoordelen, wordt de eerst ingediende oplossing uit de wachtrij (*first-in-first-out*) geselecteerd en [beoordeeld](#interpreteren-van-feedback) door een [judge](#interpreteren-van-feedback). Tijdens het beoordelen heeft een oplossing de status `Aan het uitvoeren...`.
+
+
+::: details Notitie
+
+Je kunt het beoordelingsproces van een [oplossing](#navigeren-naar-een-oplossing) niet annuleren. Terwijl een oplossing in de [wachtrij](#wachtrij) staat en terwijl de [judge](#interpreteren-van-feedback) bezig is om de oplossing te beoordelen, wordt de indienknop op de [oefeningpagina](#oefeningpagina) buiten werking gesteld en kan je geen nieuwe oplossing [indienen](#indienen-van-een-oplossing). Ook dit is een maatregel om overbelasting van het platform tegen te gaan. Nog een reden dus om te wachten met indienen tot je jezelf ervan vergewist hebt dat je oplossing geen fouten meer bevat.
+:::
+
+Zodra de [judge](#interpreteren-van-feedback) klaar is met het beoordelen van je [oplossing](#navigeren-naar-een-oplossing) krijgt ze haar finale [status](#status) en wordt de [feedbackpagina](#feedbackpagina) met gedetailleerde [feedback](#interpreteren-van-feedback) over de oplossing automatisch weergegeven in een nieuwe tab `Feedback`.
+
+![image](./student.exercise_feedback_correct_tab.png)
+
+::: tip
+
+Er zit geen beperking op het aantal keer dat je een [oplossing](#navigeren-naar-een-oplossing) kan [indienen](#indienen-van-een-oplossing) voor een [oefening](#navigeren-naar-een-oefening). Gebruik de [feedback](#interpreteren-van-feedback) die de [judge](#interpreteren-van-feedback) heeft aangeleverd om je oplossing te corrigeren of verder te verfijnen.
+:::
+
+## Navigeren naar een oplossing
```

Comment:
This should be rewritten from a different perspective. It should not focus on the ways you can navigate to a submission, but on how to see all submissions of a single exercise or see all your submissions on dodona.